dbhydra/dbhydra_core.py

```diff
@@ -442,7 +442,8 @@ def execute(self, query):
     def get_all_tables(self):
         self.cursor.execute("""SELECT table_name FROM information_schema.tables
                WHERE table_schema = 'public'""")
-        return [''.join(i) for i in self.cursor.fetchall()]
+        tables = [''.join(i) for i in self.cursor.fetchall()]
```

Comment:
usually i is used for iteration variable

Reply:
x or item is quite often used in list comprehensions

Reply:
fixed